external-docs/docs/guides/errors/build-command-errors/unknown-pydantic-error.md

```diff
@@ -1,11 +0,0 @@
-# Error - unknown pydantic validation error
```

Comment:
Please don't delete this, but can you make it clear that if the user sees this error it is deprecated and they should update to the latest version of csvcubed.

We need to make sure that people using old versions of csvcubed can still access these help/error pages.

Reply:
solved

Reply:
Not quite, this still deletes the title of the document.